man/systemctl.xml

```diff
@@ -108,6 +108,25 @@ To show all installed unit files use 'systemctl list-unit-files'.</programlistin
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><command>list-automounts</command> <optional><replaceable>PATTERN</replaceable>…</optional></term>
+
+          <listitem>
+            <para>List automount units currently in memory, ordered by mount path.  If one or more
+            <replaceable>PATTERN</replaceable>s are specified, only automount units matching one of them are shown.
+            Produces output similar to
+            <programlisting>
+WHAT        WHERE                    MOUNTED IDLE TIMEOUT UNIT
+-           /mnt/test                no      120s         mnt-test.automount
```

Comment:
Shouldn't we query the .mount unit and show WHAT for unmounted units too?

Reply:
I think we should show the mount unit belonging to the automount unit at the very least.

Reply:
Done.